pkg/backingstore/reconciler.go

```diff
@@ -1287,6 +1288,28 @@ func (r *Reconciler) updatePodTemplate() error {
 		r.PodAgentTemplate.Spec.Affinity = r.NooBaa.Spec.Affinity
 	}
 
+	// Add topology spread constraints
+	honor := corev1.NodeInclusionPolicyHonor
+	if r.PodAgentTemplate.Spec.TopologySpreadConstraints != nil {
+		r.Logger.Info("TopologySpreadConstraints already exists...")
```

Comment:
mention it is backing store in log, to make debugging easier.

Reply:
Sure, I will try to do something better. Thanks

Reply:
We cannot print the backingstore name here, as in this step we are updating the pod template itself for backingstore pod